template/src/main.js

```diff
@@ -617,6 +612,21 @@ function init () {
     $('.sidenav').find('a.nav-list-item').show();
   });
 
+  /**
+   * Executing a function after a specified amount of time
```

Comment:
Should use "Execute" instead of "Executing"

Reply:
Agree